docs/deep-dives/pnpm.md

```diff
@@ -263,7 +263,7 @@ This migration guide assumes that your project was scaffolded with a **skuba** t
       WORKDIR /workdir
     
       COPY --from=build /workdir/lib lib
-    -
+    - COPY --from=build /workdir/packages packages
```

Comment:
This shouldn't affect the majority but just adds an extra monorepo use case similar to the dev deps dockerfile changelog.